data-tool/scripts/colin_corps_extract_postgres_ddl

```diff
@@ -723,7 +723,8 @@ create table if not exists mig_corp_batch
     mig_batch_id integer     not null
         constraint fk_mig_corp_batch_mig_batch
             references mig_batch,
-    corp_num     varchar(10) not null
+    corp_num     varchar(10) not null,
+    notes            varchar(600)
```

Comment:
new `notes` field that is useful if we need to note anything specific about a corp that we are migrating.  for now we have used it for backfill test dataset to indicate which filings or scenarios we are using a given corp for